document/js-api/index.bs

```diff
@@ -1134,16 +1133,21 @@ In practice, an implementation may run out of resources for valid modules below
 <li>The maximum number of data segments defined in a module is 100000.</li>
 
 <li>The maximum number of tables, including declared or imported tables, is 1.</li>
-<li>The maximum size of a table is 10000000.</li>
 <li>The maximum number of table entries in any table initialization is 10000000.</li>
 <li>The maximum number of memories, including declared or imported memories, is 1.</li>
-<li>The initial or maximum number of pages for any memory, declared or imported, is at most 65536.</li>
 
 <li>The maximum number of parameters to any function or block is 1000.</li>
 <li>The maximum number of return values for any function or block is 1000.</li>
 <li>The maximum size of a function body, including locals declarations, is 7654321 bytes.</li>
 <li>The maximum number of locals declared in a function, including implicitly declared as parameters, is 50000.</li>
+</ul>
 
+An implementation must throw a {{RuntimeError}} if one of the following limits is exceeded during runtime:
+In practice, an implementation may run out of resources for valid modules below these limits.
+
+<ul>
+<li>The maximum size of a table is 10000000.</li>
+<li>The initial or maximum number of pages for any memory, declared or imported, is at most 65536.</li>
```

Comment:
The static sizes in memory types are already limited to that value by validation. So this could be simplified to: 
```suggestion
<li>The maximum number of pages of a memory is 65536.</li>
```
(FWIW, the reference types proposal refactored the semantics of grow in the core spec and by way of that also adds this as a runtime constraint. It was an oversight that it didn't exist before.)